src/components/HomePage/components/Steps/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import imageCalendar from './assets/image-calendar.png';
+import imageLoot from './assets/image-loot.png';
+import imageMultiChips from './assets/image-3chips.png';
+import imageChip from './assets/image-chip.png';
+import * as S from './styles';
+
+const Steps = () => {
```

Comment:
not sure about the name of this component...perhaps `Walkthrough` or something more descriptive?

Reply:
yeh good point